src/integrations/express.js

```diff
@@ -23,9 +23,9 @@ module.exports.expressJwtSecret = (options) => {
 
   return function secretProvider(req, header, payload, cb) {
     // Only RS256 is supported.
-    if (!header || header.alg !== 'RS256') {
-      return cb(null, null);
-    }
+    // if (!header || header.alg !== 'RS256') {
+    //   return cb(null, null);
+    // }
```

Comment:
If this should be removed, then it should be removed, not commented out.

Reply:
sorry for that, it was in not up to date branch I pushed first.